integration-tests/tests/aws/nino_issue_credential/nino_issue_credential-happy.test.ts

```diff
@@ -0,0 +1,235 @@
+
+    const evidence = payload.vc.evidence[0];
+    expect(evidence.type).toBe("IdentityCheck");
+    expect(evidence.strengthScore).toBe(2);
+    expect(evidence.validityScore).toBe(2);
+    expect(evidence.checkDetails[0].checkMethod).toBe("data");
+    expect(evidence.checkDetails[0].identityCheckPolicy).toBe("published");
+    expect(evidence.txn).not.toBeNull;
+
+    const credentialSubject = payload.vc.credentialSubject;
+    expect(credentialSubject.socialSecurityRecord[0].personalNumber).toBe(
+      testUser.nino
+    );
+    expect(credentialSubject.name[0].nameParts[0].type).toBe("GivenName");
+    expect(credentialSubject.name[0].nameParts[0].value).toBe(
+      testUser.firstName
+    );
+    expect(credentialSubject.name[0].nameParts[1].type).toBe("FamilyName");
+    expect(credentialSubject.name[0].nameParts[1].value).toBe(
+      testUser.lastName
+    );
+
+    expect(payload.vc.type[0]).toBe("VerifiableCredential");
+    expect(payload.vc.type[1]).toBe("IdentityCheckCredential");
+
+    expect(payload.vc["@context"][0]).toBe(
+      "https://www.w3.org/2018/credentials/v1"
+    );
+    expect(payload.vc["@context"][1]).toBe(
+      "https://vocab.london.cloudapps.digital/contexts/identity-v1.jsonld"
+    );
+
+    expect(payload.sub).not.toBeNull;
+    expect(isValidTimestamp(payload.nbf)).toBe(true);
+    expect(payload.iss).not.toBeNull;
+    expect(isValidTimestamp(payload.exp)).toBe(true);
+    expect(payload.jti).not.toBeNull;
+  });
+
+  it("should create the valid expiry date", async () => {
+    const maxJwtTtl = `/${process.env.STACK_NAME}/MaxJwtTtl`;
+    const jwtTtlUnit = `/${process.env.STACK_NAME}/JwtTtlUnit`;
+
+    const currentMaxJwtTtl = (await getSSMParamter({
+      Name: maxJwtTtl,
+    })) as any;
+
+    const currentJwtTtlUnit = (await getSSMParamter({
+      Name: jwtTtlUnit,
+    })) as any;
+
+    await ssmParamterUpdate({
+      Name: jwtTtlUnit,
+      Value: "MINUTES",
+      Type: "String",
+      Overwrite: true,
+    });
+
+    await ssmParamterUpdate({
+      Name: maxJwtTtl,
+      Value: "5",
+      Type: "String",
+      Overwrite: true,
+    });
+
+    const startExecutionResult = await executeStepFunction(
+      {
+        bearerToken: "Bearer test",
+      },
+      output.NinoIssueCredentialStateMachineArn
+    );
+
+    await ssmParamterUpdate({
+      Name: maxJwtTtl,
+      Value: currentMaxJwtTtl.Parameter.Value,
+      Type: "String",
+      Overwrite: true,
+    });
+
+    await ssmParamterUpdate({
+      Name: jwtTtlUnit,
+      Value: currentJwtTtlUnit.Parameter.Value,
+      Type: "String",
+      Overwrite: true,
+    });
+
+    const token = JSON.parse(startExecutionResult.output as any);
+
+    const [headerEncoded, payloadEncoded, signatureEncoded] =
+      token.jwt.split(".");
+
+    const payload = JSON.parse(decodeBase64(payloadEncoded));
+
+    expect(payload.exp).toBe(payload.nbf + 5 * 1000 * 60);
+  });
+
+  function decodeBase64(input: string): string {
```

Comment:
Is this needed? We can use https://developer.mozilla.org/en-US/docs/Web/API/atob to decode base64. 
> The atob() function decodes a string of data which has been encoded using Base64 encoding